src/appleseed/renderer/modeling/bsdf/bsdfsample.h

```diff
@@ -61,6 +61,8 @@ class BSDFSample
     // Roughness.
     float                           m_max_roughness;        // BSDF roughness
 
+    bool                            m_is_refraction_sample;
```

Comment:
Is this an input or an output? I suspect an output. Move it to the right section.